### PR TITLE
Reference docs for 0.25.228311.

### PR DESCRIPTION
**Pull request by DmitryVasilevsky**
**⚠ NOTE**:
This repository only contains automatically generated API documentation, and thus cannot accept pull requests.
If you would like to contribute to the API documentation in this repo, please see the [Quantum Development Kit contribution guide](https://docs.microsoft.com/azure/quantum/contributing-docs#contributing-to-the-api-references) to learn more.

### CODE REVIEW

File: api/qsharp/microsoft.quantum.arithmetic.functionwithlookuptable.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Namespace: [Microsoft.Quantum.Arithmetic](xref:Microsoft.Quantum.Arithmetic)
+
+Package: [Microsoft.Quantum.Numerics](https://nuget.org/packages/Microsoft.Quantum.Numerics)
+
+
+The return type when making a lookup table. This contains the operation that
makes the lookup table circuit, as well as all the parameters required to make
the two FixedPoint registers that need to be used as inputs and outputs to the
operator.
```
Comment (DmitryVasilevsky) [inline api/qsharp/microsoft.quantum.arithmetic.functionwithlookuptable.md:23]: Is ^M intended here?

File: api/qsharp/microsoft.quantum.arithmetic.functionwithlookuptable.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+### Apply : ([FixedPoint](xref:Microsoft.Quantum.Arithmetic.FixedPoint),[FixedPoint](xref:Microsoft.Quantum.Arithmetic.FixedPoint)) => [Unit](xref:microsoft.quantum.qsharp.valueliterals#unit-literal)  is Adj
+
+
+
+## Remarks
+
+The reason we have this return type structure is so that the operator is similar
to the other typical Q# arithmetic function implementations (a larger discussion
can had as to whether that can be changed)
```
Comment (DmitryVasilevsky) [inline api/qsharp/microsoft.quantum.arithmetic.functionwithlookuptable.md:52]: ^M ?

**Review by ricardo-espinoza — APPROVED** [review]

File: api/qsharp/microsoft.quantum.arithmetic.functionwithlookuptable.md (hunk 2 above)
Comment (DmitryVasilevsky) [reply api/qsharp/microsoft.quantum.arithmetic.functionwithlookuptable.md:52]: Will inform DOC team

File: api/qsharp/microsoft.quantum.arithmetic.functionwithlookuptable.md (hunk 1 above)
Comment (DmitryVasilevsky) [reply api/qsharp/microsoft.quantum.arithmetic.functionwithlookuptable.md:23]: Will inform DOC team

**Review by anjbur — APPROVED** [review]

**Pull request merged**